lib/fhir/core.js

```diff
@@ -556,17 +557,21 @@ module.exports = (mongo, modules) => {
               resource.meta.versionId = fhirCommon.util.generateID()
 
               const options = {
-                upsert: false, // TODO OHIE-168
+                upsert: config.getConf('operations:upserting'),
                 returnOriginal: true
               }
 
               c.findOneAndReplace({
                 id: id
-              }, resource, options, (err, result) => {
+              }, resource, options, async (err, result) => {
```

Comment:
Im not sure if this async is needed here as we dont have any promises within this function